astor/__init__.py

```diff
@@ -19,7 +21,8 @@
 from .op_util import symbol_data  # NOQA
 from .tree_walk import TreeWalk  # NOQA
 
-__version__ = '0.8.0'
+ROOT = Path(os.path.dirname(__file__))
+__version__ = (ROOT / 'VERSION').read_text().strip
```

Comment:
`read_text` came with 3.5, so this would probably break some code

Reply:
Also you can use `Path(__file__).parent` directly instead of mixing `os.path` and `pathlib` but my suggestion is instead of `pathlib` adapting this lines to `os.path` would be better (for consistency among other modules in this library)

Reply:
switched to using os.path